query/src/main/scala/filodb/query/exec/BinaryJoinExec.scala

```diff
@@ -73,9 +73,11 @@ final case class BinaryJoinExec(queryContext: QueryContext,
     val span = Kamon.currentSpan()
     val taskOfResults = childResponses.map {
       case (QueryResult(_, _, result, _, _, _), _)
-        if (result.size  > queryContext.plannerParams.joinQueryCardLimit && cardinality == Cardinality.OneToOne) =>
-        throw new BadQueryException(s"The join in this query has input cardinality of ${result.size} which" +
-          s" is more than limit of ${queryContext.plannerParams.joinQueryCardLimit}." +
+        if (
+          result.size  > queryContext.plannerParams.enforcedQuota.joinQueryCardinality &&
+          cardinality == Cardinality.OneToOne
+        ) => throw new BadQueryException(s"The join in this query has input cardinality of ${result.size} which" +
+          s" is more than limit of ${queryContext.plannerParams.enforcedQuota.joinQueryCardinality}." +
           s" Try applying more filters or reduce time range.")
```

Comment:
Where will `warnQuota` be used?

Reply:
pls, see the comment above, I will also update the description in the PR